grails-app/controllers/com/unifina/controller/api/ClusterApiController.groovy

```diff
@@ -17,45 +17,40 @@ class ClusterApiController {
 	static allowedMethods = [
 		index: "GET",
 		shutdown: "POST",
+		repair: "POST",
 	]
 
 	GrailsApplication grailsApplication
-	StreamrClient client
+	ClusterService service
```

Comment:
This variable needs to be called `clusterService`, otherwise the automatic dependency injection won't work, and it will just be `null`! @kare did it work for you?